### ISSUE TITLE
Oncall chart intermittent errors: .spec.podReplacementPolicy: field not declared in schema .status.terminating: field not declared in schema

### ISSUE THREAD

**Issue by paltaa** [grafana/helm-charts#3126]
Hey after deploying oncall chart with ArgoCD im getting these errors from time to time, any ideas on how to fix them?

First time i see something like this

`Failed to compare desired state to live state: failed to calculate diff from cache: error calculating structured merge diff: error building typed value from live resource: errors: .spec.podReplacementPolicy: field not declared in schema .status.terminating: field not declared in schema`

**Comment by freddiev4** [comment]
@paltaa yes I’m deploying with argocd. That’s the version of which chart you’re showing?

**Comment by paltaa** [comment]
hey @freddiev4 https://github.com/argoproj/argo-cd/issues/18778, looked into my grafana on call notes and it seems its an argoCD version problem, currently running argocd helm version 
`      targetRevision: 7.4.3`

**Comment by abdulrasaqmicrobridge** [comment]
Hi, any clean fix for this? currently running argocd (installed with helm chart -  v7.4.4)on k8s v1.36. Has anyone tried disabling parts of server-side diff?
